src/server/service/diagnostics/sys.rs

```diff
@@ -377,11 +377,18 @@ fn disk_hardware_info(collector: &mut Vec<ServerInfoItem>) {
     }
 }
 
+fn mac_address(mac: Option<pnet_datalink::MacAddr>) -> String {
+    match mac {
+        Some(mac) => mac.to_string(),
+        None => "none".to_string(),
+    }
+}
+
 fn nic_hardware_info(collector: &mut Vec<ServerInfoItem>) {
     let nics = pnet_datalink::interfaces();
     for nic in nics.into_iter() {
         let mut infos = vec![
-            ("mac", nic.mac_address().to_string()),
+            ("mac", mac_address(nic.mac)),
```

Comment:
When would these information useful?

Reply:
I don't know who and when will the `mac` infomation will be used.
What I know is `nic_hardware_info` will be called by `hardware_info`, and `hardware_info` will be called when you visit PD's dashboard's `/#cluster-info/host`

Reply:
/cc @breeswish can it be removed?

Reply:
They are printed in https://pingcap.com/docs-cn/stable/system-tables/system-table-cluster-hardware/ @BusyJay